exareme2/algorithms/flower/flower_data_processing.py

```diff
@@ -5,7 +5,6 @@
 from typing import Optional
```

Comment:
Could we rename this to something without the `flower` prefix?

`inputdata_preparation` or ` inputdata_preprocessing` or something similar?

WDYT?

Reply:
inputdata_preprocessing